wpimath/util/units.py

```diff
@@ -0,0 +1,171 @@
+# Copyright (c) FIRST and other WPILib contributors.
+# Open Source Software; you can modify and/or share it under the terms of
+# the WPILib BSD license file in the root directory of this project.
+
+from math import degrees, radians, pi
+
+kInchesPerFoot = 12.0
+kMetersPerInch = 0.0254
+kSecondsPerMinute = 60
+kMillisecondsPerSecond = 1000
+kKilogramsPerLb = 0.453592
+
+
+def metersToFeet(meters: float) -> float:
+    """Converts given meters to feet.
+
+    :param meters: The meters to convert to feet.
+
+    :returns: Feet converted from meters.
+    """
+    return metersToInches(meters) / kInchesPerFoot
+
+
+def feetToMeters(feet: float) -> float:
+    """Converts given feet to meters.
+
+    :param feet: The feet to convert to meters.
+
+    :returns: Meters converted from feet.
+    """
+    return inchesToMeters(feet * kInchesPerFoot)
+
+
+def metersToInches(meters: float) -> float:
+    """Converts given meters to inches.
+
+    :param meters: The meters to convert to inches.
+
+    :returns: Inches converted from meters.
+    """
+    return meters / kMetersPerInch
+
+
+def inchesToMeters(inches: float) -> float:
+    """Converts given inches to meters.
+
+    :param inches: The inches to convert to meters.
+
+    :returns: Meters converted from inches.
+    """
+    return inches * kMetersPerInch
+
+
+def degreesToRadians(degrees: float) -> float:
+    """Converts given degrees to radians.
+
+    :param degrees: The degrees to convert to radians.
+
+    :returns: Radians converted from degrees.
+    """
+    return radians(degrees)
+
+
+def radiansToDegrees(radians: float) -> float:
+    """Converts given radians to degrees.
+
+    :param radians: The radians to convert to degrees.
+
+    :returns: Degrees converted from radians.
+    """
+    return degrees(radians)
+
+
+def radiansToRotations(radians: float) -> float:
+    """Converts given radians to rotations.
+
+    :param radians: The radians to convert.
+
+    :returns: rotations Converted from radians.
+    """
+    return radians / (pi * 2)
```

Comment:
Use `math.tau` instead.

I'd also personally question the usefulness of the rotation/angle conversions here; I think doing the arithmetic directly would be clearer than having a function, since tau already exists as a constant.

Reply:
> I'd also personally question the usefulness of the rotation/angle conversions here;

@auscompgeek and @virtuald

Is it better to use Python's `math` instead of creating a separate module specific to measure conversion?
Because I don't know if this makes it easier for new people that will use these functions.
If the answer is yes, I'll close this pull request and I could use the `math` directly on the [statespaceflywheel example.](https://github.com/wpilibsuite/allwpilib/blob/main/wpilibjExamples/src/main/java/edu/wpi/first/wpilibj/examples/statespaceflywheel/Robot.java)

Reply:
I think it would be useful to have the conversions that aren't in math here. For things that are in math, I think an alias is appropriate:

```
 degreesToRadians = math.radians
 ```

Reply:
Okay, I think aliasing the math functions is reasonable too, assuming it shows up in the docs correctly.